model-optimizer/install_prerequisites/install_prerequisites.sh

```diff
@@ -72,7 +72,7 @@ if [[ $DISTRO == "centos" ]]; then
     fi
 elif [[ $DISTRO == "ubuntu" ]]; then
     sudo -E apt update
-    sudo -E apt -y install python3-pip python3-venv libgfortran3
+    sudo -E apt -y install python3-pip python3-venv libgfortran5
```

Comment:
> libgfortran*

@artanokhov @asuhov Do we really need to expose this dependency directly? (which OV component requires that?)

Reply:
libgfortran3 package was required to install MXNet python module for model optimizer